src/components/Report/index.js

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
+import './index.css';
 
 const Report = props => {
     return (
-        <div className="single-raport">
+        <div
+            className={`single-raport ${props.active && 'active'}`}
```

Comment:
It is a good practice to control UI state of existing components with Redux. Why? When deciding whether to use Redux or not, we should get only "one source of truth", and minimize usage of setState functionality. Good job.

For further read:

https://medium.com/javascript-scene/10-tips-for-better-redux-architecture-69250425af44

especially refer to point number 5.